libs/features/webapp-prompt/webapp-prompt.css

```diff
@@ -182,6 +182,10 @@
     transform: scaleX(0) translateZ(0);
   }
 
+  [dir = "rtl"] .appPrompt-progress {
+    transform-origin: top right;
```

Comment:
Anything we need to keep in mind in regards to animations with this?

Reply:
I don't think so. By increasing `scaleX` from `0` to `1` the loader expands to fill the space. By starting the animation on the right instead of the left, it has nowhere to expand but to the left. 
AFAICT this doesn't affect any other animations and it applies only in rtl domains.
Lmk if I've misunderstood your question.